packages/react-signer/src/snap.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017-2023 @polkadot/react-signer authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import * as snap from 'azero-wallet-adapter';
+
+import { keyring } from '@polkadot/ui-keyring';
+
+export const connectSnap = async (): Promise<void> => {
+  try {
+    await snap.connect();
+
+    // TODO: Set the RPC URL based on the current network
+    await snap.setRpcUrl({ rpcUrl: 'https://rpc.test.azero.dev/' });
```

Comment:
To be set using currently selected RPC in the app

Reply:
@piotr-roslaniec it should be updated before merging, right? Do you need a help with that?

Reply:
Yes, and yes. I guess I could hack around this with `polkadot.js` but I was wondering whether there is some already implemented method to fetch the current network (like some sort of React hook).

Reply:
Do you need a url string or a connected polkadot-js api object? You can use `settings` from `@polkadot/ui-settings` to get the url string

packages/apps/src/Root.tsx:46